draft-ietf-mls-protocol.md

```diff
@@ -2447,26 +2436,48 @@ and administered under a Specification Required policy {{!RFC8126}}.
 
 ## MLS Ciphersuites
 
-The "MLS Ciphersuites" registry lists identifiers for suites of cryptographic
-algorithms defined for use with MLS.  These are two-byte values, so the maximum
-possible value is 0xFFFF = 65535.  Values in the range 0xF000 - 0xFFFF are
-reserved for vendor-internal usage.
+A cipher suite value defines a combinaison of a protocol version
+and of the set of cryptographic algorithms used for this version MLS.
 
-Template:
+Cipher suite names follow the naming convention:
 
-* Value: The two-byte identifier for the ciphersuite
-* Name: The name of the ciphersuite
-* Reference: Where this algorithm is defined
-
-The initial contents for this registry are as follows:
-
-| Value  | Name                    | Reference |
-|:-------|:------------------------|:----------|
-| 0x0000 | P256_SHA256_AES128GCM   | RFC XXXX  |
-| 0x0001 | X25519_SHA256_AES128GCM | RFC XXXX  |
+~~~
+   CipherSuite MLS_LVL_KEM_AEAD_HASH_SIG = VALUE;
+~~~
 
-[[ Note to RFC Editor: Please replace "XXXX" above with the number assigned to
-this RFC. ]]
+| Component | Contents |
+|:----------|:---------|
+| MLS       | The string "MLS" followed by the major and minor version, e.g. "MLS10" |
+| LVL       | The security level |
+| KEM       | The KEM algorithm used for HPKE in TreeKEM group operations |
+| AEAD      | The AEAD algorithm used for HPKE and message protection |
+| HASH      | The hash algorithm used for HPKE and the MLS KDF |
+| SIG       | The Signature algorithm used for message authentication |
+
+This specification defines the following cipher suites for use with MLS 1.0.
+
+|          Description                                |    Value    |
+|:----------------------------------------------------|:------------|
+| MLS10_128_HPKEX25519_AES128GCM_SHA256_Ed25519       | {0x00,0x01} |
+| MLS10_128_HPKEP256_AES128GCM_SHA256_P256            | {0x00,0x02} |
+| MLS10_128_HPKEX25519_CHACHA20POLY1305_SHA256_Ed25519| {0x00,0x03} |
+| MLS10_256_HPKEX448_AES256GCM_SHA384_Ed448           | {0x00,0x04} |
+| MLS10_256_HPKEP521_AES256GCM_SHA384_P521            | {0x00,0x05} |
+| MLS10_256_HPKEX448_CHACHA20POLY1305_SHA256_Ed448    | {0x00,0x06} |
+
+The KEM/DEM constructions used for HPKE are defined by {{HPKE}}.
+The corresponding AEAD algorithms AEAD_AES_128_GCM and AEAD_AES_256_GCM, are
+defined in {{RFC5116}}. AEAD_CHACHA20_POLY1305 is defined
+in {{RFC7539}}. The corresponding hash algorithms are defined in {{!SHS}}.
+
+The mandatory-to-implement ciphersuite for MLS 1.0 is
+`MLS10\_128\_HPKE25519\_AES128GCM\_SHA256\_Ed25519` which is using
```

Comment:
Shouldn't the mandatory suite be P-256? X25519 has never been mandatory

Reply:
> Shouldn't the mandatory suite be P-256?

Why?

> X25519 has never been mandatory

It has always been so far

Reply:
As  discussed at the last interim, we keep this MTI.